### PR TITLE
Pr vtol hitl

### PR DESCRIPTION
**Pull request by RomanBapst**
- adds a HITL model for standard vtol

### CODE REVIEW

**Comment by LorenzMeier** [comment]
CI failure?

**Comment by TSC21** [comment]
@RomanBapst was this tested with jMAVSim only or did you have a shot with Gazebo too?

**Comment by RomanBapst** [comment]
Something with the mixers, I'll check

**Comment by hamishwillee** [comment]
@RomanBapst When this goes in, can we please have an update to the HITL docs [here](https://dev.px4.io/en/simulation/hitl.html)?

Specifically, update the list of HITL compatible airframes and confirm that the instructions are correct for this case.

**Comment by RomanBapst** [comment]
@hamishwillee Sure!

**Comment by TSC21** [comment]
Jenkins test this please.

**Comment by TSC21** [comment]
@RomanBapst can you review why CI is failing? I can't trigger the builds (no write permissions).

**Comment by RomanBapst** [comment]
@dagar Could you also check what's wrong here with CI please?

**Comment by TSC21** [comment]
Can we reboot CI here @dagar?

**Comment by dagar** [comment]
@RomanBapst this doesn't fit in fmu-v2.

File: ROMFS/px4fmu_common/init.d/2200_standard_vtol_hitl [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#
+
+sh /etc/init.d/rc.vtol_defaults
+
+if [ $AUTOCNF == yes ]
+then
+	param set BAT_N_CELLS 3
+	param set CAL_ACC0_ID 1376264
+	param set CAL_ACC0_XOFF 0.01
+	param set CAL_ACC0_XSCALE 1.01
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/2200_standard_vtol_hitl]: Why are you setting the calibration parameters?

Comment (RomanBapst) [reply ROMFS/px4fmu_common/init.d/2200_standard_vtol_hitl]: @dagar This is probably coming from the SITL startup file. I'll go over the params and kick out the unnecessary ones.

**Comment by RomanBapst** [comment]
@dagar I addressed your comments. Good to go from my side.

File: ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!nsh
+#
+# @name Standard vtol HITL (gazebo standard_vtol model)
```
Comment (hamishwillee) [inline ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil]: Shouldn't this be "Standard VTOL HITL" ?

Comment (hamishwillee) [reply ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil]: Might be better as:
```
@name Standard VTOL HITL (Gazebo only)
```

File: ROMFS/px4fmu_common/mixers/standard_vtol_hitl.main.mix [hunk 3]
```diff
@@ -0,0 +1,26 @@
+Mixer for standard vtol plane (SITL) with motor x configuration
+=========================================================
+
+This file defines a single mixer for a standard vtol plane (SITL gazebo) with motors in X configuration. The plane has two ailerons and one elevator.
```
Comment (hamishwillee) [inline ROMFS/px4fmu_common/mixers/standard_vtol_hitl.main.mix]: Everywhere:
- VTOL not vtol 
- This config is referred to as a quadplane in our docs. 
- do you mean SITL or HITL? 

So possibly you mean:
```
Mixer for Standard VTOL (QuadPlane) with motor x configuration (Gazebo HITL)
=========================================================

This file defines a single mixer for a Standard VTOL QuadPlane (HITL Gazebo) with motors in X configuration. The plane has two ailerons and one elevator.
```

File: ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil (hunk 2 above)
Comment (hamishwillee) [reply ROMFS/px4fmu_common/init.d/1002_standard_vtol.hil]: Actually, the existing HIL frames have names like HIL ...., so perhaps this should be named 
"HIL Standard VTOL QuadPlane"

**Comment by hamishwillee** [comment]
Has this stalled?

**Comment by dagar** [comment]
It hit flash limits on fmu-v2 previously. It might currently be okay after a rebase.

**Comment by dagar** [comment]
Rebased on master. Good to go if CI passes.

**Comment by RomanBapst** [comment]
@hamishwillee Thanks, I addressed you last comments. Can be merged after checks pass.

**Comment by hamishwillee** [comment]
Thanks @RomanBapst  - Yes, thanks for fixes. I did minor update to docs and merged: https://github.com/PX4/Devguide/pull/519

**Pull request merged**